### ISSUE TITLE
URIs of requirements classes, requirements, conformance classes, and abstract tests

### ISSUE THREAD

**Issue by ghobona** [opengeospatial/ogcapi-connected-systems#28]
The OGC policy document [OGC-NA Name type specification – specification elements (10-103r1)](https://docs.ogc.org/pol/10-103r1.html) requires that: 

The end of the URI of a requirements class be of the form `/req/<req-class-name>`

The end of the URI of requirement be of the form `/req/<req-class-name>/<req-name>`

The end of the URI of a conformance class be of the form `/conf/<conf-class-name>`

The end of the URI of requirement be of the form `/conf/<conf-class-name>/<conf-name>`

**Using modifiers**

Note that you can use hyphens as a modifier, for example `/req/earth-science/bedrock-geology`

**Comment by alexrobin** [comment]
Thanks @ghobona. I think that's what we are doing for requirements. Is there a specific place where the requirement URI is not conformant?

**Comment by alexrobin** [comment]
Ah ok I modeled these on OGC API - Features.
I will change them to keep only 3 segments.
Thanks

**Comment by alexrobin** [comment]
This should be fixed in [latest draft of part 1](https://opengeospatial.github.io/ogcapi-connected-systems/api/part1/standard/23-001r0.html)

**Comment by ghobona** [comment]
@alexrobin I can confirm that the problem of the number of segments in the URIs have been resolved.

**Issue 28B**

However, I also noticed that some permissions and recommendations begin with a `/req/` segment.

URI segments of recommendations should begin `/rec/`.

URI segments of recommendations should begin `/per/`.

**Issue 28C**

I also noticed that the requirements classes were not linked to the requirements. So I created a pull request showing how to create the link for one requirements class.

https://github.com/opengeospatial/ogcapi-connected-systems/pull/29/files
